app/src/display/widgets/Kconfig

```diff
@@ -22,5 +22,12 @@ config ZMK_WIDGET_OUTPUT_STATUS
     default y if BT
     select LVGL_USE_LABEL
     select LVGL_FONT_MONTSERRAT_16
+    
+config ZMK_WIDGET_WPM_STATUS
+    bool "Widget for displaying typed words per minute"
+    default y
```

Comment:
Should this be a default widget for OLEDs? Seems like this would be more of an optional feature.

Reply:
Agreed this should be opt-in.

Reply:
Done

Reply:
To make this not default, this `default y` line should be removed, which then will use the implicit `default n` from Kconfig.